contact.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <title>Contact Me</title>
+</head>
+
+<body>
+    
+
+        <div class="top-page">
+            <h1 id="name-head">Trevor Stikeleather</h1>
+            <ul class="header-links">
```

Comment:
on every page I would wrap this url in a nav (or anything div, etc.) tag to make it easier to float right.
Then I also have some hints on the styling to make your header style simpler